### ISSUE TITLE
Replace ':' with '/' as default port separation syntax for Tcl addbot syntax

### ISSUE THREAD

**Issue by vanosg** [eggheads/eggdrop#476]
Make using /s for port separation the default documented method for IPv4 and IPv6, still leaving the current : support in code to maintain backwards compatibility. See #470 for background.

**Comment by maimizuno** [comment]
The data in the userfile (by way of GETUSER) uses spaces as delimiters. Should (and could?) the syntax allow that, so it's the same when written as when it's read? Otherwise, script parsing would requiire an seemingly CPU-wasteful SPLIT/REGSUB for no other good reason.

\<Domino> ## getuser Bot2 botaddr
\<Bot> Bot..2.address 49178 49178

**Comment by Cizzle** [comment]
Would be slightly more consistent with the `+bot` and `chaddr` partyline commands. Whatever is decided (and I think related previous discussion favoured spaces), best to be consistent all over. Would that be for 1.9 or 2.0?

**Comment by vanosg** [comment]
Agreed, spaces would standardize a bit more. If we wanted to pursue this, I would propose:
- Accepting spaces or :s in 1.8.x (remove /s all together, as we haven't published the /s as a standard in 1.8 yet)
- Making spaces default in 1.9.0, still allow :s
- Removing :s in 2.0

**Comment by maimizuno** [comment]
Vanosg's solution would be the most backward-compatible, forward-thinking, and TCL/userfile-cross-compatible option.

**Comment by maimizuno** [comment]
THE huge changelog posted the other day shows that this is still ana ctive item being merged into the 1.8.3 code; this conflicts with another entry which supercedes this ans uses spaces only (with back*-compat* options of colon/slash).

Will this be spaces, or colon/slash? (If so, can this entry [#476] go away?)

**Comment by vanosg** [comment]
I don't think there is anything to do on this for 1.9; spaces are already documented as 'the way' via #496 and :'s are documented as marked as deprecated. The world is at peace. Moving this task to 2.0
